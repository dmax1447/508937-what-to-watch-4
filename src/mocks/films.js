
const films = [
  {
    id: `f01`,
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    picture: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    id: `f02`,
    title: `Bohemian Rhapsody`,
    picture: `bohemian-rhapsody.jpg`,
  },
  {
    id: `f03`,
    title: `Macbeth`,
    picture: `macbeth.jpg`,
  },
  {
    id: `f04`,
    title: `Aviator`,
    picture: `aviator.jpg`,
  },
  {
    id: `f05`,
    title: `We need to talk about Kevin`,
    picture: `we-need-to-talk-about-kevin.jpg`,
  },
  {
    id: `f06`,
    title: `What We Do in the Shadows`,
    picture: `what-we-do-in-the-shadows.jpg`,
  },
  {
    id: `f07`,
    title: `Revenant`,
    picture: `revenant.jpg`,
  },
  {
    id: `f08`,
    title: `Johnny English`,
    picture: `johnny-english.jpg`,
  },
];

export default films;
