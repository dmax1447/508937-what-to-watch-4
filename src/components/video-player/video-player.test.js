import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

import films from '../../mocks/tests/films.js';

it(`<VideoPlayer /> should render properly`, () => {
  const player = renderer
        .create(<VideoPlayer movie={films[0]} />)
        .toJSON();

  expect(player).toMatchSnapshot();
});
