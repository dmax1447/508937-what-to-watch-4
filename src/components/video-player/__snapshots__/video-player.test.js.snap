// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoPlayer /> should render properly 1`] = `
<div
  className="small-movie-card__image"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <img
    alt="Fantastic Beasts: The Crimes of Grindelwald"
    height="175"
    src="fantastic-beasts-the-crimes-of-grindelwald.jpg"
    width="280"
  />
</div>
`;
